recipes/fgpyo/meta.yaml

```diff
@@ -48,4 +48,5 @@ extra:
   recipe-maintainers:
     - tfenne
     - nh13
-    - NatPRoach
+  identifiers:
```

Comment:
this needs to be under the extras section

Reply:
@bgruening I think it already is!